projects/v2/src/app/components/organ-table-selector/organ-table-selector.component.ts

```diff
@@ -72,10 +107,14 @@ export class OrganTableSelectorComponent implements OnInit {
         });
       });
     });
-    this.hasSomeOrgans = this.selection.selected.length > 0;
+    // this.hasSomeOrgans = this.selection.selected.length > 0;
```

Comment:
* Remove commented out code
* Remove unnecessary console.log

Reply:
Removed and all urls have been put in place.